tests/unit/back/models/user.test.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+
+var chai = require('chai');
+var expect = chai.expect;
+var User = require('../../../../').models.User;
+var Promise = require('bluebird');
+
+require('../../settings');
+
+var user = new User({
+  username: 'test',
+  email: 'email@test.com',
+  password: '$2a$10$Bd4I9p6xSuphD3gSPk006.fMR5/Dr9LjJD7JhvJcm7hpDaS0wJruq'
```

Comment:
Since the password is hashed, it would be nice to write the actual password in a comment